File: P1A5/main (3).cs
//ANA ALEJANDRA PARRA
//ING. TI.
//0595316
//FECHA c#
using System;

class Fecha 
{
    public int mes;
    public int dia;
    public int anio;

    public Fecha(int m, int d, int a) 
    {
        mes = m;
        dia = d;
        anio = a;
    }

    public void MostrarFecha() 
    {
        Console.WriteLine(mes + "/" + dia + "/" + anio);
    }
}

class FechaTest 
{
    static void Main() 
    {
        
        Fecha miFecha = new Fecha(8, 30, 2026);

        Console.Write("La fecha es: ");
        miFecha.MostrarFecha();
    }
}

File: P1A5/main (3).py
#ANA ALEJANDRA PARRA
#ING. TI.
#0595316
#FECHA PYTHON

class Fecha:
    
    def __init__(self, m, d, a):
        self.mes = m
        self.dia = d
        self.anio = a

    def mostrarFecha(self):
        print(str(self.mes) + "/" + str(self.dia) + "/" + str(self.anio))


mi_fecha = Fecha(8, 30, 2026)

print("La fecha es: ", end="")
mi_fecha.mostrarFecha()
